README.rst

```diff
@@ -44,11 +44,585 @@ Available Features
 Dependencies
 ============
 
-* Python 3.4 or higher
-* Django 2.0 or higher
```

Comment:
can we take advantage of this update to update the screenshots? For example, it would be nice to show the infinite scroller, if it doesn't take too much time, could you create a gif with the action of scrolling a huge ipv6 subnet?
I think it's a cool feature we should promote.

If you see any other outdated screenshot, or any other outdated part of the README which needs improvement, please fix it.

Reply:
Okay, I added a GIF, do you prefer this GIF or a cropped version of it only focusing on infinite scroll? :smile:

Reply:
Will write in chat